tests/gis_tests/test_geoforms.py

```diff
@@ -374,11 +374,21 @@ class PointForm(forms.Form):
 class GeometryWidgetTests(SimpleTestCase):
 
     def test_get_context_attrs(self):
-        """The Widget.get_context() attrs argument overrides self.attrs."""
+        # geom_type gets transfered in the returned context
+        widget = BaseGeometryWidget(attrs={'geom_type': 'POLYGON'})
+        context = widget.get_context('point', None, None)
+        self.assertEqual(context['geom_type'], 'Polygon')
+
+        # attrs argument overrides self.attrs
         widget = BaseGeometryWidget(attrs={'geom_type': 'POINT'})
         context = widget.get_context('point', None, attrs={'geom_type': 'POINT2'})
         self.assertEqual(context['geom_type'], 'POINT2')
 
+        # Geometry geom_type gets transfered properly in the returned context
+        widget = BaseGeometryWidget(attrs={'geom_type': 'GEOMETRY'})
+        context = widget.get_context('point', None, None)
```

Comment:
Likewise, it seems odd to keep `'point'` here:
```suggestion
        context = widget.get_context('geometry', None, None)
```